definitions/infra-apacheserver/definition.yml

```diff
@@ -1,2 +1,6 @@
 domain: INFRA
 type: APACHESERVER
+goldenTags: []
```

Comment:
goldenTags are optional, so if they are empty we could avoid adding the field directly (but I also don't see any harm in this)

Reply:
I did this when they where private definitions to quickly find the ones that don't have any golden tag defined. So devs could scan the project and fix that. We can remove it it's annoying though

Reply:
If anyone ever complains it's easy to remove so 👍